_sass/module/_landing-description.scss

```diff
@@ -0,0 +1,17 @@
+.landing__description__header {
+  @include heading('h3');
+  color: $color-grey-2;
```

Comment:
I moved this to the `heading()` mixin, I think all `h3`'s should be this color, right?